protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -557,6 +557,50 @@ definitions:
       - overwrite
       #- upsert_dedup # TODO chris: SCD Type 1 can be implemented later
       - append_dedup # SCD Type 1 & 2
+  OAuth2Specification:
```

Comment:
There have been so many iterations of this so my memory could be wrong, but with the new OAuth engine, isn't all the OAuth stuff in the protocol not needed anymore? Why is it being brought back?

Reply:
You are right we don't need it anymore. This is mostly an act of equalising the files since they've drifted so I don't run into any odd pieces in case something is using it when I switch everything back to the V0 protocols and remove this file (which is V1).

Reply:
kk. can you mark it as deprecated then, please? even just a comment would be great.